DRAFT.md

```diff
@@ -591,24 +599,24 @@ become valueless after it has been moved from.
 `allocator_traits<Allocator>::value_type` is not the same type as `T`, the
 program is ill-formed. Every object of type `indirect<T, Allocator>` uses an
 object of type `Allocator` to allocate and free storage for the owned object as
-needed. The owned object is constructed using the function
-`allocator_traits<allocator_type>::construct` and destroyed using the function
-`allocator_traits<allocator_type>::destroy`.
+needed.
 
-// DRAFTING NOTE: [indirect.general]#3 modeled on [container.reqmts]#64
+3. Constructing an owned object using an allocator `a` and arguments `args...`
```

Comment:
All the uses in the constructors below say "with arguments ..., using the allocator `alloc`" but this has the two terms the other way around. I think the way you used it later is better, so should change this here:

```suggestion
3. Constructing an owned object with arguments `args...` using an allocator `a` 
```